src/main/java/com/google/api/generator/gapic/model/HttpBindings.java

```diff
@@ -63,6 +65,8 @@ public int compareTo(HttpBinding o) {
 
   public abstract String pattern();
 
+  public abstract List<String> additionalPatterns();
```

Comment:
Do we need to use `List` here? Can we use `Set`? Does order matter? Do duplicates matter?

Reply:
I think technically duplicates are allowed even though they will be useless. Bindings come in list from proto file descriptor, so keeping them as list here.